server/sonar-server/src/main/java/org/sonar/ce/monitoring/CEQueueStatus.java

```diff
@@ -134,5 +84,5 @@
   /**
    * Time spent processing batch reports since startup.
    */
-  long getProcessingTime();
+  long getProcessingTimeInMs();
```

Comment:
I don't think the `InMs` is useful. Is `Date.getTime()` named `Date.getTimeInMs()` ?
In addition, `addError` and `addSuccess` take a long which is a time in ms